pkg/querier/queryrange/queryrangebase/results_cache_test.go

```diff
@@ -751,9 +752,11 @@ func TestResultsCache(t *testing.T) {
 			Cache: cache.NewMockCache(),
 		},
 	}
-	rcm, _, err := NewResultsCacheMiddleware(
+	c, err := cache.New(cfg.CacheConfig, nil, log.NewNopLogger())
+	require.NoError(t, err)
+	rcm, err := NewResultsCacheMiddleware(
 		log.NewNopLogger(),
-		cfg,
+		c,
```

Comment:
All I see here is changes to the test to satisfy the new signature, have there been any tests added to confirm that empty responses are actually cached? Sorry if I just missed them by only looking at changes.

Reply:
Yes there is a new test file for this. This is the current test file for metric result.